### DISCUSSION TITLE
RX-V685 works - but no input selection possible.

### DISCUSSION THREAD

**Discussion by MichaelHeimann** [mvdwetering/yamaha_ynca#276] [General]
Hi, just wanted to post that my RX-V685 works for some features. 

![image](https://github.com/mvdwetering/yamaha_ynca/assets/3189562/7b61692a-4e11-433c-b963-014a81c19cd0)

I get a Warning in the Logs saying:

```
Logger: ynca.enums
Source: /usr/local/lib/python3.12/site-packages/ynca/enums.py:215
First occurred: 9:57:05 AM (1 occurrences)
Last logged: 9:57:05 AM

Unknown value 'AUDIO5' in Input
```

And I cannot switch the input (or couldn't find out how - maybe this is related to the error message?

**Comment by mvdwetering** [comment]
You should be able to switch inputs in the mediaplayer. In the image you posted tap the mediaplayer and in the popup there should be a box for input selection. The intention is that input switching should still work (except for the unknown input), even with the error, but maybe that is not handled properly.

Could you submit an issue for the unknown AUDIO5 input with the requested diagnostics?

**Comment by MichaelHeimann** [comment]
Ah! Didn't realize the First "button" is a mediaplayer that offers more options in a pop-up. Thanks!

I'll open the issue for AUDIO5.
